Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,14 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
```

Comment:
quick note, you dont need to import this css file because you already imported it on a higher order component